include/Core/Base/Mixin/Fetcher.php

```diff
@@ -36,17 +36,22 @@ protected function get_post_statuses() {
 	/**
 	 * Get the list of post types by post status and count.
 	 *
+	 * @param string $feature Feature of post.
```

Comment:
Add `@since xxx Added $feature  param` phpdoc comment.

I like the way you are passing the feature as a parameter.

Reply:
Fixed.